src/fides/api/schemas/tcf.py

```diff
@@ -29,7 +29,13 @@ class NonVendorSection(UserSpecificConsentDetails):
     systems: List[EmbeddedVendor] = []  # Systems that use this TCF attribute
 
 
-class TCFPurposeConsentRecord(NonVendorSection, MappedPurpose):
+class CommonPurposeFields(FidesSchema):
+    """Fields shared between the purpose sections of the TCF Experience"""
+
+    retention_period: Optional[str]
```

Comment:
no you're right—they don't show up in the Purpose tab. I put them in Purpose because I thought that was the only way to get them into EmbeddedPurpose but there's probably another way...

Reply:
OK great! let's update it downstream to just put them into EmbeddedPurpose - I think it's going to cause problems per some of my other comments.